src/Validation/Validator.php

```diff
@@ -2051,22 +2313,61 @@ protected function _canBeEmpty($field, $context)
     /**
      * Returns true if the field is empty in the passed data array
      *
-     * @param mixed $data value to check against
+     * @param mixed $data Value to check against.
      * @return bool
+     * @deprecated Use isEmpty() instead
      */
     protected function _fieldIsEmpty($data)
     {
-        if (empty($data) && !is_bool($data) && !is_numeric($data)) {
+        return $this->isEmpty($data, self::EMPTY_ALL);
```

Comment:
Can we please use static in these cases?
Especially ALL can at some point be needed as LTB to add another missing one into it.